File: Hashtable_Flutter/lib/model/Table.dart
import 'Node.dart';

class Table {
  List<Node?> _nodes = [];

  Table(size){
    _nodes = List<Node?>.filled(size, null);
  }
  void setNodes(List<Node> nodes) => _nodes = nodes;

  List<Node?> getNodes() => _nodes;
}

File: Hashtable_JavaSwing/src/main/java/tn/usousse/eniso/ia1/stage/example/model/Table.java
package tn.usousse.eniso.ia1.stage.example.model;



public class Table{
    public Table() {
    }

    private Node[] nodes;

    public Node[] getNodes() {
        return nodes;
    }

    public void setNodes(Node[] nodes) {
        this.nodes = nodes;
    }

    public Table(int size) {
        nodes  = new Node[size];
    }
}
